src/inventory_ui.cpp

```diff
@@ -1882,12 +1874,14 @@ inventory_input inventory_selector::get_input()
     res.action = ctxt.handle_input();
     res.ch = ctxt.get_raw_input().get_first_input();
 
-    std::pair<point, bool> ct_pair = ctxt.get_coordinates_text( w_inv );
-    if( ct_pair.second ) {
-        point p = ct_pair.first;
-        res.entry = find_entry_by_coordinate( p );
-        if( res.entry != nullptr ) {
-            return res;
+    cata::optional<point> o_p = ctxt.get_coordinates_text( w_inv );
+    if( o_p ) {
+        point p = o_p.value();
+        if( window_contains_point_relative( w_inv, p ) ) {
+            res.entry = find_entry_by_coordinate( p );
+            if( res.entry != nullptr ) {
+                return res;
+            }
```

Comment:
I haven't tested, but judging from the backtrace the crash might happen when you click on a category heading, which doesn't contain any items and would fail the assert in `item_location::any_item`.

Reply:
Yes. This is it.
![iTOOLS](https://user-images.githubusercontent.com/1931904/88265755-1af10d80-ccdf-11ea-94f2-fc47dfc9262b.png)

Reply:
Can you please test again? I think I've resolved the issue.

Reply:
Cannot reproduce a SIGABRT.

But I cannot select items in inventory in this inv. view mode if I changed the view mode (`;`) before selecting them. Clicking items on the left side does nothing:
![unselectable items](https://user-images.githubusercontent.com/1931904/88702078-ec7a9480-d11b-11ea-869f-382a780d9791.png)

I can select not worn items in another inv. view mode. But if I change view mode twice they are shown without the `+` sign.
![no selection indicator](https://user-images.githubusercontent.com/1931904/88702912-12ecff80-d11d-11ea-97f6-88d4e086ea31.png)